### PR TITLE
Chai date and time assertions

### PR DESCRIPTION
**Pull request by mguterl**
I wrote some custom chai assertions and did a quick writeup of how and why.

### CODE REVIEW

**Comment by mguterl** [comment]
@mitchlloyd @rockwood @superchris @cdmwebs or any other JavaScript dudes, I would appreciate some feedback on this.

File: posts/2013-07-02-chai-date-time-assertions.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+  return this.assert(
+    actual == expected,
+    'expected ' + this._obj + ' to equal ' + time,
+    'expected ' + this._obj + ' to not equal ' + time
+  );
+});
+
+chai.Assertion.addChainableMethod('equalDate', function(date) {
+  var expectedYear  = date.getUTCFullYear(),
```
Comment (mitchlloyd) [inline posts/2013-07-02-chai-date-time-assertions.md]: I think this stuff can be simplified using date.toDateString()

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toDateString

Comment (mguterl) [inline posts/2013-07-02-chai-date-time-assertions.md]: Thanks @mitchlloyd I incorporated this change and it works!

**Comment by cdmwebs** [comment]
:metal:

**Pull request merged**